test/test_tensorexpr.py

```diff
@@ -441,20 +505,35 @@ def test_abs(x, y):
         test_ceil,
         test_trunc,
         test_abs,
+        test_log,
+        test_log2,
+        test_log10,
+        test_log1p,
+        test_rsqrt,
+        test_exp,
+        test_expm1,
+        test_erf,
+        test_erfc,
+        test_frac,
+        test_lgamma,        
+        test_sigmoid,
+        test_reciprocal,
+        test_neg,
+        test_relu,
     }
-    rand_a = torch.rand(1024, dtype=float)
-    rand_b = torch.rand(1024, dtype=float)
-    zeros = torch.zeros(1024, dtype=float)
+    rand_a = torch.rand(1024, dtype=torch.float)
```

Comment:
@bertmaher @resistor  I had to change type to torch.float to make sure doubles were not being used. This caused fp discrepencies in many of the instrinsic functions for which I had to relax rtol and atol for numpy.allclose. Let me know what you guys think in terms of bitwise inaccuracy